app/components/contributor-list/template.hbs

```diff
@@ -1,41 +1,45 @@
-<div id="{{get-section-id layout.title}}" class="contributor-list-container p-t-xl p-b-xl"
-     style="{{containerStyle}}">
-    <div class="container">
-        {{#if layout.title}}
-            <div class="row text-center section-title">
-                <h2>
-                    {{layout.title}}
-                </h2>
+{{#if layout.title}}
```

Comment:
this can be simplified more, no need for so many conditional if we use a computed on the model.... pr is too big as is, so will make a separate pr. This component should appear the same as it did before as it is now though.